Formula/q.rb

```diff
@@ -1,30 +1,30 @@
-class Q < Formula
-  include Language::Python::Virtualenv
+# frozen_string_literal: true
 
+class Q < Formula
   desc "Run SQL directly on CSV or TSV files"
   homepage "https://harelba.github.io/q/"
-  url "https://github.com/harelba/q/archive/2.0.20.tar.gz"
-  sha256 "46793aef623aac3700856c699cc04810b7a53533f829318729cee900c499a7e1"
-  license "GPL-3.0-or-later"
-  head "https://github.com/harelba/q.git", branch: "master"
+  url "https://github.com/harelba/q/archive/v3.1.6.tar.gz"
+  sha256 "e63ba4ae49647f764c5255ad7065d2c614fdf03a2f7349a795de69529701fab8"
 
-  bottle do
-    sha256 cellar: :any_skip_relocation, arm64_monterey: "c72ca06a7c9dbe3b3eaee1b8db72811edbf7e64fbee5b694bcf2ed7e8d877d50"
-    sha256 cellar: :any_skip_relocation, arm64_big_sur:  "c72ca06a7c9dbe3b3eaee1b8db72811edbf7e64fbee5b694bcf2ed7e8d877d50"
-    sha256 cellar: :any_skip_relocation, monterey:       "4511c183df36704ec7cb497b4a319409875ea2ef6068255ae2a2e0a2d7293e29"
-    sha256 cellar: :any_skip_relocation, big_sur:        "4511c183df36704ec7cb497b4a319409875ea2ef6068255ae2a2e0a2d7293e29"
-    sha256 cellar: :any_skip_relocation, catalina:       "4511c183df36704ec7cb497b4a319409875ea2ef6068255ae2a2e0a2d7293e29"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:   "bed14a331133ff96b85fa37e0729ca695bd273f78ee82e792185d137edf9917a"
-  end
```

Comment:
Please don't touch the bottle block.

Reply:
i thought that this since this was a new version, then it needs to be removed (and added automatically again by the framework). I'll readd it.